src/components/Hub/FilterButton.tsx

```diff
@@ -1,24 +1,54 @@
 import React, { useState } from "react"
 import styled from "@emotion/styled"
 import { Button } from "@material-ui/core"
-import { FilterCategory } from "../../redux/hub/hub.types"
+import { FilterCategory, FilterCategoryName } from "../../redux/hub/hub.types"
+
+type FilterButtonColors = {
+  [category in FilterCategoryName]: {
+    backgroundColor: string
+    color: string
+  }
+}
+
+const filterButtonColors: FilterButtonColors = {
```

Comment:
In #329, I had worked this in to theme https://github.com/jina-ai/dashboard/pull/329/files#diff-0e21fb1c9519a397050defc2148a2d1b966a0982cc4cf196e1e6b007f8d095e5R78

Let's have a chat about what works best.